setup.py

```diff
@@ -32,8 +38,7 @@
     url="https://github.com/pyro-ppl/numpyro",
     author="Uber AI Labs",
     install_requires=[
-        "jax>=0.2.11",
-        "jaxlib>=0.1.62",
+        f"jax[cpu]{_jax_version_constraints}",
```

Comment:
Do you know what happens if `jax`, `jaxlib` GPU version is available and we want to update NumPyro
```
pip install -U numpyro
```
Will CPU versions replace the available GPU versions?

Reply:
This is indeed what happens (provided there was a change in the jax version NumPyro depends on). Updates would need to also specify the cuda option (and the jax release repo URL) in order not to replace with the CPU version.
One option would be to not install jaxlib unless the user specifies the target platform (which apparently what the folks at jax are going for). This means that jaxlib does not get replaced unintentionally but carries the risk of ending up with updated versions of numpyro and jax and an outdated (incompatible) jaxlib from the initial installation..

Reply:
So I think we can remove this `cpu` option and set
```
f"jax>={_jax_version_constraints}",
"jaxlib>=0.1.62",
```
as previously. Since the next release, we will relax `jax/jaxlib` version - when people use numpyro in colab, they won't need to worry about jax/jaxlib installation (colab environment already has those versions)